src/components/vaForm/template.tsx

```diff
@@ -0,0 +1,234 @@
+import { VaForm as VaFormType } from './formatted-type'
+
+type VaFormProps = VaFormType
+
+export function VaForm({
+  title,
+  formName,
+  formNumber,
+  revisionDate,
+  issueDate,
+  formType,
+  benefitCategories,
+  administration,
+  formUrl,
+  toolUrl,
+  toolIntro,
+  usage,
+  linkTeasers,
+  relatedForms,
+}: VaFormProps) {
+  const formatDate = (dateString: string) => {
+    return new Date(dateString).toLocaleDateString('en-US', {
+      year: 'numeric',
+      month: 'long',
+    })
+  }
+
+  return (
+    <main className="va-l-detail-page">
+      <div className="usa-grid usa-grid-full">
+        <div className="usa-width-three-fourths">
+          <article className="usa-content">
+            <h1 className="vads-u-margin-bottom--0">{title}</h1>
+
+            <dl>
+              <div className="vads-u-margin-bottom--4">
+                <dt className="va-introtext">
+                  <dfn className="vads-u-visibility--screen-reader">
+                    Form name:
+                  </dfn>
+                  {formName}
+                </dt>
+              </div>
+
+              {(revisionDate || issueDate) && (
+                <div>
+                  <dd>
+                    <dfn className="vads-u-font-weight--bold vads-u-display--inline">
+                      Form revision date:
+                    </dfn>
+                    {` ${formatDate(revisionDate || issueDate!)}`}
+                  </dd>
+                </div>
+              )}
+
+              <div className="vads-u-margin-y--1">
+                <dd>
+                  <dfn className="vads-u-font-weight--bold vads-u-display--inline">
+                    Related to:
+                  </dfn>
+                  {formType === 'employment' && ` Employment or jobs at VA`}
+                  {formType === 'non-va' && (
+                    <>
+                      {` A non-VA form. For other government agency forms, go to the `}
+                      <a href="https://www.gsa.gov/reference/forms">
+                        GSA forms library
+                      </a>
+                      .
+                    </>
+                  )}
+                  {(!formType || formType === 'benefit') && (
+                    <>
+                      {benefitCategories && benefitCategories.length > 0
+                        ? ` ${benefitCategories.join(', ')}`
+                        : ` ${administration || 'VA benefits and services'}`}
+                    </>
+                  )}
+                </dd>
+              </div>
+            </dl>
+
+            {usage && (
+              <>
+                <h2 className="vads-u-margin-top--4">When to use this form</h2>
+                <div dangerouslySetInnerHTML={{ __html: usage }} />
```

Comment:
This isn't a `<Paragraph>`, so... 🤷 I thought there was a component we used for these formatted texts? I don't remember it, though.